dts/bindings/arc/arc,xccm.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2017, synopsy
```

Comment:
copyright doesn't look correct, year, and name.

Reply:
Done, thanks for your check.